c/flake.nix

```diff
@@ -0,0 +1,53 @@
+{
+  description = "Templates for getting started with Aux";
+
+  inputs.nixpkgs.url = "github:auxolotl/nixpkgs/nixos-unstable";
+
+  outputs =
+    { self, nixpkgs }:
+    let
+      forAllSystems =
+        function:
+        nixpkgs.lib.genAttrs [
+          "x86_64-linux"
+          "aarch64-linux"
+          "x86_64-darwin"
+          "aarch64-darwin"
+          "i686-linux"
+          "mipsel-linux"
+          "powerpc64le-linux"
+        ] (system: function nixpkgs.legacyPackages.${system});
+    in
+    rec {
+      devShells = forAllSystems (pkgs: {
+        default = pkgs.mkShell {
+          hardeningDisable = [ "fortify" ];
+          inputsFrom = pkgs.lib.attrsets.attrValues packages;
+        };
+      });
+
+      packages = forAllSystems (pkgs: rec {
```

Comment:
Can we move the package defection to a file called default.nix

Reply:
> Can we move the package defection to a file called default.nix

Is there a need to split the flake? it is quite small

Reply:
It allows it to have old nix compatibility. that’s the main thing for me. Also prevents redeclaring the package a few times.